llvm/cmake/modules/AddSecurityFlags.cmake

```diff
@@ -135,6 +135,9 @@ macro(append_common_extra_security_flags)
   # explicitly requested
   if (NOT CMAKE_POSITION_INDEPENDENT_CODE)
     message(FATAL_ERROR "To enable all necessary security flags, CMAKE_POSITION_INDEPENDENT_CODE must be set to ON")
+  else()
+    include(CheckPIESupported)
```

Comment:
should this go upstream?

Reply:
This whole file doesn't exist in upstream. That's our own customization that addresses our own SDL requirements.

We can talk with the community about adopting some of those requirements, but that's a separate topic which will be huge. Like there are tons of warnings caused by `-Wconversion` that is required per our processes.

Reply:
ah ok sorry, i was deceived by the directory :)